ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-ospf.j2

```diff
@@ -88,22 +88,37 @@
 
 #### Router OSPF Router Redistribution
 
-| Process ID | Source Protocol | Route Map |
-| ---------- | --------------- | --------- |
+| Process ID | Source Protocol | Include Leaked | Route Map |
+| ---------- | --------------- | -------------- | --------- |
 {%         for process_id in router_ospf.process_ids | arista.avd.natural_sort('id') %}
 {%             if process_id.redistribute is arista.avd.defined %}
 {%                 set source_protocols = [] %}
 {%                 if process_id.redistribute.connected is defined %}
-{%                     do source_protocols.append(('connected', process_id.redistribute.connected.route_map | arista.avd.default('-'))) %}
+{%                     if process_id.redistribute.connected.include_leaked is arista.avd.defined(true) %}
+{%                         set include_leaked = 'enabled' %}
+{%                     else %}
+{%                         set include_leaked = 'disabled' %}
+{%                     endif %}
```

Comment:
```suggestion
{%                     set include_leaked = 'enabled'  if process_id.redistribute.connected.include_leaked is arista.avd.defined(true) else 'disabled' %}
```

Reply:
Thanks for the review. I did it this way for consistency with the current template. Should I refactor the whole template with your suggested approach ?

Reply:
Just focus on the part you are adding for now.

As a side note, I was almost suggesting to refactor the different `if` statements for `connected`, `static` and `bgp` in a for loop as this template looks quite bloated and creating a tuple to then access the various element with `[0]`,... does not look very optimal.

Reply:
I really don't like the loops, when the data models are dicts in the schema. They are kept as dicts because there are different options under each (in EOS).

Reply:
Re the suggestion from @gmuloc, we try to avoid that style everywhere in Jinja. It will probably also trigger errors from j2lint.

Reply:
not for these 3 regarding the loop but ok for the other not implemented options so "for future cases" let's keep it like this.
```
Leaf1(config-router-ospf)#show cli commands | grep redistribute
[no|default] redistribute ( ( rip | aggregate ) ) [ route-map MAPNAME ]
[no|default] redistribute ( ( static | connected | bgp ) ) [ include leaked ] [ route-map MAPNAME ]
[no|default] redistribute isis [ include leaked ] [ level-1|level-2|level-1-2 ] [ route-map MAPNAME ]
[no|default] redistribute ospf ( ( instance [ include leaked ] ) | leaked ) [ match ( internal | external | nssa-external ) ] [ route-map MAPNAME ]
```

Regarding the suggestion - we write five lines that could be one 3 times so that makes the templates much longer and less easy to maintain. j2lint does not error on this syntax (if there were some operators in the line it could eventually complain about some spaces but it is ok with this and it is good as it is very valid syntax).

All that being said - let's leave it like this. Going to resolve my comment